creator/Documents/ScoreboardIntroduction.md

```diff
@@ -149,46 +149,45 @@ To get a list of all of the players in a world (including any fake ones you crea
 Use this command with the name of a player to get a list of that player's tracked objectives.
 
 ```
-/scoreboard players list <player>
+/scoreboard players list [playername: target]
 ``` 
 
 ### operation
 
 You can perform mathematical operations to calculate and assign scores.
 
 ```
-/scoreboard players operation Player1 objective operation Player2 objective
+/scoreboard players operation <TargetPlayer> <TargetObjective> <operation> <SourcePlayer> <SourceObjective>
 ```
 
-Each operation uses the scores for two players.  The players can be from two different objectives, which is why you have to specify the objective for each player.
+Each operation uses the scores for two players. The players can be from two different objectives, which is why you have to specify the objective for each player.
 
-For each operation, the score for Player2 is used to get a result, and the result is returned as a new score for Player1. If that isn't super clear, here is a whole tutorial to show you how operations work: [Scoreboard Operations Tutorial](ScoreboardOperationsTutorial.md).
+For each operation, the score for SourcePlayer is used to get a result, and the result is returned as a new score for TargetPlayer. If that isn't super clear, here is a whole tutorial to show you how operations work: [Scoreboard Operations Tutorial](ScoreboardOperationsTutorial.md).
 
-- **`%=` Modulo** - 
-Divides the first score by the second score and returns the remainder.
+- **`%=` Modulo** - Divides the first score by the second score and returns the remainder.
 
 - **`*=` Multiplication** - Returns the product of the scores after multiplying them.
 
 - **`+=` Addition** - Returns the sum of the scores.
 
 - **`-=` Subtraction** - Returns the value you get when you subtract the second score from the first.
 
-- **`/=` Division** - Returns the number you get when you divide the first score by the second score.
+- **`/=` Division** - Returns the number you get when you divide the first score by the second score. The returned value is rounded down.
```

Comment:
Added a little note of the flooring behavior in case it brought up ambiguity in how decimals are handled.